kubetest2/kubetest2-gce/deployer/deployer_helpers.go

```diff
@@ -0,0 +1,26 @@
+package deployer
+
+import (
+	"fmt"
+	"os"
+)
+
+func (d *deployer) setKubernetesPathIfNotSet() error {
+	if d.kubernetesRootPath == "" {
```

Comment:
if d.kubernetesRootPath != "" {
   return nil
}

will avoid nested { one layer and also easier to read.

Reply:
Done.